final_task/rss_reader/Entry.py

```diff
@@ -0,0 +1,67 @@
+import logging
+from Logging import logging_decorator
+import html
+
+
+class Entry:
+    """class for every article from http:link...link.rss"""
+    @logging_decorator
+    def __init__(self, title: str = "", date: str = "", article_link: str = "", summary: str = "", links: list = []):
```

Comment:
During our course we have explained why is it bad to set mutable objects as default arguments. Please, fix it

Reply:
fixed